docs/admin/managing_servers.md

```diff
@@ -1,43 +1,42 @@
-TripleA can be played over the internet and has a big online community.
-For tasks like this we have a couple servers running, which power the lobby, the forum and a dozen bots.
+Admin doc for managing the TripleA Lobby and Bots.
+
+## Lobby
 
-## The Lobby
-The Lobby is hosted on the same server as the legacy [tripleawarclub.org](https://tripleawarclub.org) forum, as well as our [dice server](https://dice.tripleawarclub.org).
 We use `systemd` to start and stop those services.
 ```
 sudo service triplea-lobby start|stop|status|restart
 ```
+
 All TripleA related processes are started as the `triplea` user, so if you ever get an error because of insufficient permissions, make sure all the files are owned by the `triplea` user.
 
-The command is pretty much self-explanatory, but be advised, that restarting the lobby quits all connections to all bots.
+Be advised, restarting the lobby quits all connections to all bots.
 Even if the lobby restarts, the bots won't reconnect automatically, they will have to be restarted on their own.
 
-_Of course not everyone is permitted to perform such an critical operation. This command is restricted to users with full sudo rights._
+_This command is currently restricted to users with full sudo rights._
+
+## [The Dice server](https://github.com/triplea-game/dice-server)
 
-## The Dice server
-Dice servers are used to ensure integrity of dice rolls for Play-by-E-Mail games in order to prevent cheating.
-The code can be found at [our Repository](https://github.com/triplea-game/dice-server), the dice server is written in PHP.
 To restart the dice server just run
 ```
 sudo service nginx restart
 ```
 on the tripleawarclub.org/dice.tripleawarclub.org server.
 
-## The Forums
-The new forum runs on the NJ server (forums.triplea-game.org), it's powered by NodeBB and written in Node-JavaScript.
-It uses a mongo database.
-NodeBB is deployed via git and dependencies are installed using npm.
+## [NodeBB Forums](https://forums.triplea-game.org)
+- Runs on the "NJ" linode server (as of 2017-07-12) 
+- Powered by NodeBB and written in Node-JavaScript with a mongo database.
+- NodeBB is deployed via git and dependencies are installed using npm. (TODO: add exact commands to deploy NodeBB)
```

Comment:
https://docs.nodebb.org/ has all the information about NodeBB, no need to write our own!